packages/react-components/src/components/FileUploadCards.tsx

```diff
@@ -30,7 +30,7 @@ export interface FileUploadCardsProps {
  * @beta
  */
 export const FileUploadCards = (props: FileUploadCardsProps): JSX.Element => {
-  const truncateLength = 15;
+  const truncateLength = 12;
```

Comment:
How are we chosing this number. Also, can't we instead use CSS `overflow:hidden text-overflow:ellipses overflow-wrap:anywhere` here?

Reply:
Oh yesss. Completely forgot about that CSS property Haha